zerver/lib/event_queue.py

```diff
@@ -233,13 +254,15 @@ def to_dict(self):
 
     @classmethod
     def from_dict(cls, d):
+        # type: (Dict[str, Any]) -> EventQueue
         ret = cls(d['id'])
         ret.next_event_id = d['next_event_id']
         ret.queue = deque(d['queue'])
         ret.virtual_events = d.get("virtual_events", {})
         return ret
 
     def push(self, event):
+        # type: (Dict[str, Any]) -> None
```

Comment:
Maybe that won't work because we are appending `event` to `self.queue` which is of type `deque[Dict[str, Any]]`. We can't append a `MutableSequence` to a deque of `Dict`s.